react/react1/week2/todo/src/TodoItem.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+//{ description, deleteItem, id } is from the parent todolist component
+export function TodoItem({ description, deleteItem, id }) {
```

Comment:
```suggestion
export function TodoItem({ item, onItemDeleted }) {
```

I think it makes more sense to pass the whole item rather than piece by piece.
```js
<TodoItem
  item={todo}
  onItemDeleted={deleteItem}
  key={todo.id}
/>
```

`key` is just a detail of rendering an array so I tend to put it at the bottom of the list of props since it's not data that the component uses.

Reply:
good suggestion :)